locales/volto.pot

```diff
@@ -2387,6 +2409,11 @@ msgstr ""
 msgid "Origin"
 msgstr ""
 
+#: components/manage/LinksToItem/LinksToItem
+# defaultMessage: Overview and management of relations of all content items
```

Comment:
Will the addition of this message string require an update to the screen shots?

Is a user truly able to manage relations from this panel, or only look at an overview? If the latter:

```suggestion
# defaultMessage: Overview of relations of all content items
```

...and update all instances of the message.

Reply:
> Will the addition of this message string require an update to the screen shots?

The button linking to the relations control panel is only for site managers. Therefore I prefer not to mention this in the user documentation.

Reply:
> Is a user truly able to manage relations from this panel, or only look at an overview?

He/she is indeed able to **modify** relations. That's the bonus of the new relations control panel. But as the main intent of this content menu addition is about providing a view with the backlinks/references to this page (content item) **in text**, and not with custom relations, the reduction to 'overview' is fine.